docs/configuration.rst

```diff
@@ -1,16 +1,108 @@
+
+The table below lists the available images and the Confluent software packages they contain.  You'll note that some images are identified as ```cp-enterprise-${component_name}```.   These images include proprietary components that must be licensed from Confluent when deployed.
+
++------------------+------------------------------+--------------+-----------------------------------------+
+| Component        | Image Name                   | Type         | Packages Included                       |
++==================+==============================+==============+=========================================+
+| Base Image       | cp-base                      | Open Source  | - zulu-openjdk-8                        |
++------------------+------------------------------+--------------+-----------------------------------------+
+| Kafka            | cp-kafka                     | Open Source  | - confluent-kafka-*                     |
++------------------+------------------------------+--------------+-----------------------------------------+
+| Kafka            | cp-enterprise-kafka          | Enterprise   | - confluent-kafka-*                     |
+|                  |                              |              | - confluent-rebalancer                  |
+|                  |                              |              | - confluent-support-metrics             |
++------------------+------------------------------+--------------+-----------------------------------------+
+| Control Center   | cp-enterprise-control-center | Enterprise   | - confluent-control-center              |
++------------------+------------------------------+--------------+-----------------------------------------+
+| Replicator       | cp-enterprise-replicator     | Enterprise   | - confluent-kafka-replicator            |
+|                  |                              |              | - confluent-schema-registry             |
+|                  |                              |              | - confluent-control-center              |
++------------------+------------------------------+--------------+-----------------------------------------+
+| Kafka Connect    | cp-kafka-connect             | Enterprise   | - confluent-kafka-connect-jdbc          |
+|                  |                              |              | - confluent-kafka-connect-hdfs          |
+|                  |                              |              | - confluent-schema-registry             |
+|                  |                              |              | - confluent-control-center              |
+|                  |                              |              | - confluent-kafka-connect-elasticsearch |
```

Comment:
`confluent-kafka-connect-s3` should be in here too. technically `confluent-kafka-connect-storage-common` is in there as well, but that's probably not worth highlighting for users as it is required for both hdfs and s3

Reply:
added `confluent-kafka-connect-s3`